pkg/constants/constants.go

```diff
@@ -19,6 +19,7 @@ const (
 	PluginKind        = "Plugin"
 	ManifestExtension = ".yaml"
 	KrewPluginName    = "krew" // plugin name of krew itself
+	ManifestURL       = "https://github.com/rewanth1997/kubectl-fields/raw/master/deploy/krew/plugin.yaml"
```

Comment:
These constants are for production code only. I think `testutil_test.go` would be a better place for this.

Reply:
Done